build.gradle

```diff
@@ -1,7 +1,8 @@
 buildscript {
-    ext.android_plugin_version = '3.3.2'
+    ext.android_plugin_version = '3.4.0'
     ext.kotlin_version = '1.3.21'
     ext.jacoco_version = '0.8.1'
+    ext.navigation_version = '1.0.0-beta01'
```

Comment:
It looks like `2.0.0` is stable with AndroidX? Should we be using that instead?

Reply:
I think this just got updated recently. I will update to the latest.

Reply:
i'm seeing the IDE recommend '1.0.0' so i'll stick with that for now